.NET/Microsoft.Recognizers.Text.Sequence/Extractors/BasePhoneNumberExtractor.cs

```diff
@@ -124,11 +124,16 @@ public override List<ExtractResult> Extract(string text)
                             }
                         }
 
+                        ers.Remove(er);
```

Comment:
Can you add a code comment here to describe what types of entities would fall here?